src/renderer/src/hooks/useImportAction.ts

```diff
@@ -75,7 +76,7 @@ export const useImportAction = (
         throw new Error(t('errors.invalid'))
       }
 
-      const fixedText = data.text.trim().replace(/[^a-zA-Z0-9 ]/g, '') // Remove all special characters except spaces
+      const fixedText = UtilsHelper.removeSpecialCharacters(data.text.trim(), true)
```

Comment:
In most cases the `allowSpaces` is true, I'd set this property with default value true. I'd add the `trimText` property in this method, like this:

```typescript
type TRemoveSpecialCharacterOptions = {
  allowSpaces?: boolean
  trimText?: boolean
}

...

static removeSpecialCharacters(text: string, options?: TRemoveSpecialCharacterOptions = {}) {
  options = { allowSpaces: true, trimText: false, ...options }
  text = text.replace(options.allowSpaces ? /[^a-zA-Z0-9 ]/g : /[^a-zA-Z0-9]/g, '')

  if (options.trimText) text = text.trim()

  return text
}
```